src/elements/fields/DropdownMultiField.tsx

```diff
@@ -109,13 +110,16 @@ export default function DropdownMultiField({
 
   const labels = servar.metadata.option_labels;
   const labelMap: Record<string, string> = {};
-  let options;
+  let options: any[] = [];
 
-  if (dynamicOptions.length > 0) {
-    options = dynamicOptions.map((option) => {
-      labelMap[option.value] = option.label;
-      return { value: option.value, label: option.label };
-    });
+  if (servar.metadata.salesforce_sync && !editMode) {
+    if (loadingDynamicOptions) options = [];
```

Comment:
idk if the loadingDynamicOptions needs to be returned from the hook. You could just check this internal to the hook and return `[]` for the options if it's true